src/monte_carlo.h

```diff
@@ -0,0 +1,75 @@
+#ifndef ONECHIPML_MONTE_CARLO_RL
+#define ONECHIPML_MONTE_CARLO_RL
```

Comment:
No need to fix anything, this is just a question, why does you header guards ends with RL ?

Reply:
For reinforcement learning (to distinguish this method from the monte carlo estimate also in the library)